hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/RaftInvocationContext.java

```diff
@@ -109,7 +111,7 @@ public void updateMember(CPMember member) {
 
             CPMembersContainer newContainer = new CPMembersContainer(currentContainer.version, newMembers);
             if (membersContainer.compareAndSet(currentContainer, newContainer)) {
-                logger.info("Replaced " + existingMember + " -> " + member);
+                logger.info("Replaced " + existingMember + " with " + member);
```

Comment:
should we move this log to `debug` level?

Reply:
sure, but not now. after we see CP persistence tests are passing without any issues...